Assets/src/PlayerWeapons.cs

```diff
@@ -68,6 +68,7 @@ public class PlayerWeapon {
   public Sprite Sprite => Weapon.Sprite;
   public Sprite InHandSprite => Weapon.InHandSprite;
   public AudioClip ShootSound => Weapon.ShootSound;
+  public bool isReloading = false;
```

Comment:
We use PascalCase for class fields. It's probably not what most C# projects are using nowadays, but best to keep it consistent.

Reply:
Actually, on second thought, would it be possible to remove `isReloading` and use the existing `WeaponReload.InProgress()`  method instead?